code/modules/surgery/organs/lungs.dm

```diff
@@ -0,0 +1,309 @@
+/obj/item/organ/internal/lungs
+	name = "lungs"
+	icon_state = "lungs"
+	parent_organ = "chest"
+	slot = "lungs"
+	organ_tag = "lungs"
+	gender = PLURAL
+	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
But you can fit a book, and I don't think Cultists haul around pocket tomes. Also, of course you could... if you folded it up; provided this isn't good for the lung, but that's just specifics.